src/NzbDrone.Core/Extras/Subtitles/SubtitleFile.cs

```diff
@@ -13,15 +13,19 @@ public SubtitleFile()
 
         public Language Language { get; set; }
 
-        public string AggregateString => Language + LanguageTagsAsString + Extension;
+        public string AggregateString => Language + Title + LanguageTagsAsString + Extension;
+
+        public int Copy { get; set; }
 
         public List<string> LanguageTags { get; set; }
 
+        public string Title { get; set; }
+
         private string LanguageTagsAsString => string.Join(".", LanguageTags);
 
         public override string ToString()
         {
-            return $"[{Id}] {RelativePath} ({Language}{(LanguageTags.Count > 0 ? "." : "")}{LanguageTagsAsString}{Extension})";
+            return $"[{Id}] {RelativePath} ({Language}{(Title is not null ? "." : "")}{Title ?? ""}{(LanguageTags.Count > 0 ? "." : "")}{LanguageTagsAsString}{Extension})";
```

Comment:
I think doing this with a string builder will be more readable in the long run, especially with the chained conditionals.